src/jit/emitxarch.cpp

```diff
@@ -4462,7 +4542,39 @@ void emitter::emitIns_R_R_R_I(
     id->idReg2(reg1);
     id->idReg3(reg2);
 
+    code_t code = 0;
+
+    switch (ins)
+    {
+        case INS_pextrb:
+        case INS_pextrd:
+        case INS_pextrq:
+        case INS_pextrw_sse41:
+        case INS_extractps:
+        case INS_vextractf128:
+        case INS_vextracti128:
```

Comment:
The only delta from `R_R_I` is that `INS_shld` and `INS_shrd` aren't included. They should never be hit though since this is only called for AVX instructions.